Source/WebCore/platform/audio/IIRFilter.cpp

```diff
@@ -201,16 +201,18 @@ double IIRFilter::tailTime(double sampleRate, bool isFilterStable)
     input[0] = 1;
 
     // Process the first block and get the max magnitude of the output.
-    process(input.span().first(AudioUtilities::renderQuantumSize), output.span());
-    magnitudes[0] = VectorMath::maximumMagnitude(output.data(), AudioUtilities::renderQuantumSize);
+    auto inputSpan = input.span().first(AudioUtilities::renderQuantumSize);
+    auto outputSpan = output.span().first(AudioUtilities::renderQuantumSize);
+    process(inputSpan, output.span());
```

Comment:
Why is this using `output.span()` instead of `outputSpan`?

Reply:
My bad, will fix.